libs/core/src/components/pds-popover/pds-popover.tsx

```diff
@@ -0,0 +1,134 @@
+import { Component, Element, Host, Listen, h, Prop, State } from '@stencil/core';
+import { PlacementType } from '@utils/types';
+
+@Component({
+  tag: 'pds-popover',
+  styleUrl: 'pds-popover.scss',
+  shadow: true,
+})
+export class PdsPopover {
+  /**
+   * Reference to the Host element
+   */
+  @Element() el: HTMLPdsPopoverElement;
+
+  /**
+   * Determines when the popover is active
+   * @defaultValue false
+   */
+  @State() active = false;
+
+  /**
+   * Determines the action that triggers the popover
+   * @defaultValue "show"
+   */
+  @Prop() popoverTargetAction: 'show' | 'toggle' = 'show';
```

Comment:
I think we can add a bit of context here to explain that this only works if the popover type is set to manual.